posthog/batch_exports/http.py

```diff
@@ -283,11 +283,18 @@ def validate_destination(self, destination_attrs: dict):
             config = destination_attrs["config"]
             # for updates, get the existing config
             self.instance: BatchExport | None
+            view = self.context.get("view")
+
             if self.instance is not None:
                 existing_config = self.instance.destination.config
+            elif view is not None and "pk" in view.kwargs:
+                # Running validation for a `detail=True` action.
+                instance = view.get_object()
+                existing_config = instance.destination.config
```

Comment:
Not sure why `self.instance` is `None`, but it is.

Reply:
Maybe somebody with more Django knowledge can enlighten me: I would assume that an action with `detail=True` would populate the serializer's `instance`.

Reply:
hmm yeah that is strange